src/naf-dialog-adapter.js

```diff
@@ -33,14 +33,24 @@ const PC_PROPRIETARY_CONSTRAINTS = {
 const ICE_RECONNECT_INTERVAL = 2000;
 const INITIAL_ROOM_RECONNECTION_INTERVAL = 2000;
 
+const WEBCAM_SIMULCAST_ENCODINGS = [
+  { scaleResolutionDownBy: 4, maxBitrate: 500000 },
+  { scaleResolutionDownBy: 2, maxBitrate: 1000000 },
+  { scaleResolutionDownBy: 1, maxBitrate: 5000000 }
+];
+
+// Used for simulcast screen sharing.
+const SCREEN_SHARING_SIMULCAST_ENCODINGS = [{ dtx: true, maxBitrate: 1500000 }, { dtx: true, maxBitrate: 6000000 }];
+
 const isFirefox = navigator.userAgent.toLowerCase().indexOf("firefox") > -1;
 
 export default class DialogAdapter {
   constructor() {
     this._timeOffsets = [];
     this._occupants = {};
     this._micProducer = null;
-    this._videoProducer = null;
+    this._cameraProducer = null;
```

Comment:
Is there a reason this was split up? Seems like we (currently) are only allowing producing one or the other. Might simplify the code a bit to keep it as `_vidoeProducer` so you don't have to check both in multiple places.

Reply:
I guess now that we are going to add support for webcam avatars it will make sense to separate this?

Reply:
yeah, this is another thing the "groups" concept I talked about will be helpful for. No strong preference on if we keep the 2 producer variables separate or combine for this PR.

Reply:
ok, let's keep it like it is and address groups when we implement multiple producers support.